docs/resources/useful-resources.md

```diff
@@ -1,5 +1,7 @@
 # Useful Jupyter Accessibility Resources
 
+Looking for [accessibility docs?](../accessibility-docs)
```

Comment:
Wherever I removed the links to the a11y toolbar or the NVDA add-on, I added a link to `accessibility-docs`

Reply:
Same comment as above